src/helper/client.cpp

```diff
@@ -196,10 +196,21 @@ bool client::handle_command(network::request_init::request &command)
     auto response = std::make_shared<network::request_init::response>();
     try {
         auto res = context_->publish(std::move(command.data));
-        if (res && res->valid()) {
-            response->verdict = "record";
-            response->triggers = std::move(res->data);
-            response->actions = std::move(res->actions);
+        if (res) {
+            switch (res->type) {
+            case engine::action_type::block:
+                response->verdict = "block";
```

Comment:
Not a big issue but this string literals could be constants

Reply:
Fair enough